rollup.config.js

```diff
@@ -0,0 +1,11 @@
+import pkg from './package.json';
+
+export default [
+	{
+		input: 'index.js',
+		external: ['ms'],
```

Comment:
What is this 'ms'?

Reply:
It was in the rollup template, may be specific to rollup, but don't really know what it is

Reply:
I think I can delete, will sqaush out